src/block/styled-components.js

```diff
@@ -34,7 +34,11 @@ function build(breakpoints: BreakpointsT) {
         value.forEach((v, index) => {
           // Do not create a media query for the smallest breakpoint.
           if (index === 0) {
-            styles[property] = transform(v);
+            // styletron will throw when value is undefined. if so, do not apply rule
+            const transformedValue = transform(v);
+            if (transformedValue !== undefined) {
+              styles[property] = transform(v);
```

Comment:
There are a couple more usage of `transform` below. How about adding this check in the transform function itself?

Reply:
Oh, I see, it's different for every props.

Reply:
thank you 🙏